examples/calibration_flash.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+import cv2
+import depthai as dai
+import argparse
+
+
+# Connect Device
+calibJsonFile = str((Path(__file__).parent / Path('models/depthai_calib.json')).resolve().absolute())
+
+parser = argparse.ArgumentParser()
+parser.add_argument('calibJsonFile', nargs='?', help="Path to V6 calibration file in json", default=calibJsonFile)
+args = parser.parse_args()
+
+print("The following code is going to replace the calibration data stored on your device. Proceed with Caution.")
+input("Press Enter to continue... or Press ctrl + c to abort")
```

Comment:
Would it be possible to make a backup to a file of the existing calibration data?

Reply:
it can be but Since these are used for test where should I write it to ? Create a folder under examples and store it there ?

Reply:
I was thinking just as a simple ...old.calib in the working directory (with a message printed that old calib was saved there), as they are also in the .gitignore. More in the idea for users to be able to go back, if the new calib doesn't give better results. But anyway, to keep the example simple, maybe we can add that instead to calibrate.py on depthai repo.